docker-compose.yaml

```diff
@@ -0,0 +1,12 @@
+services:
+  tinymod:
+    build: 
+      context: .
+      dockerfile: Dockerfile.dev
+      args:
+        - WORKDIR=/usr/src/tinymod
+    volumes:
+    - ./tinymod:/usr/src/tinymod/tinymod
+    ports:
+    - 8000:8000
```

Comment:
Why does port 8000 need to be open?

Reply:
tinymod/plugins/voice.py looked like it used it for you to send requests to make it speak? Could be very wrong on that, but kinda why I was thinking might be going the admin page route.

Reply:
That's outgoing port only, and the tts code is very temporary right now, so I think its fine if we just don't open any ports.